src/DropDownMenu/DropDownMenu.js

```diff
@@ -288,7 +312,7 @@ class DropDownMenu extends Component {
           <div
             style={prepareStyles(Object.assign({}, styles.label, open && styles.labelWhenOpen, labelStyle))}
           >
-            {displayValue}
+            {displayLeftIcon}{displayValue}{displayRightIcon}
```

Comment:
@oliviertassinari A little bit late, but I finally moved it to separate function